src/commands/sarif/upload.ts

```diff
@@ -83,6 +83,19 @@ export class UploadSarifReportCommand extends Command {
   public async execute() {
     enableFips(this.fips || this.fipsConfig.fips, this.fipsIgnoreError || this.fipsConfig.fipsIgnoreError)
 
+    // remove this notice in April 2025
```

Comment:
please mark this as a `TODO`, so that it's easier to find, *and* has an owner associated with it (which I'm guessing is you).

 ```suggestion
    // TODO(julien): remove this notice in April 2025
```